luma/core/device.py

```diff
@@ -208,8 +212,12 @@ def __init__(self, device=None, **kwargs):
         assert self.bits_per_pixel in image_converters, f"Unsupported bit-depth: {self.bits_per_pixel}"
         self.__image_converter = image_converters[self.bits_per_pixel]
 
+        self.framebuffer = framebuffer or diff_to_previous(num_segments=16)
         self.capabilities(width, height, rotate=0, mode="RGB")
 
+        path = f"/dev/fb{self.id}"
+        self.__fp = open(path, "wb")
```

Comment:
even though it's private, I'd prefer a more descriptive name for fp

Reply:
fixed